src/components/HeaderPageLayout.tsx

```diff
@@ -19,6 +19,9 @@ type HeaderPageLayoutProps = ChildrenProps &
         /** The background color to apply in the upper half of the screen. */
         backgroundColor?: string;
 
+        /** TestID to apply to the whole section container */
+        testID?: string;
```

Comment:
Same here, let's make this required:

```suggestion
        testID: string;
```